vpn_slice/__main__.py

```diff
@@ -51,6 +51,17 @@ def get_default_providers():
             dns = DNSPythonProvider or DigProvider,
             hosts = PosixHostsFileProvider,
         )
+    elif platform.startswith('freebsd'):
+        from .mac import BSDRouteProvider
+        from .linux import ProcfsProvider
```

Comment:
FreeBSD `/proc` works just like Linux, in terms of being able to look up PPID and executable name?

Reply:
Not exactly, but it stopped throwing errors and worked therefore

But i will quickly implement the changes needed to support the freebsd procfs